mokapot/model.py

```diff
@@ -543,7 +519,7 @@ def load_model(model_file):
         LOGGER.info("Loading the Percolator model.")
 
         weight_cols = [c for c in weights.index if c != "m0"]
-        model = Model(estimator=LinearSVC(), scaler="as-is")
+        model = Model(estimator=LinearSVC(), scaler=StandardScaler())
         weight_vals = weights.loc[weight_cols]
         weight_vals = weight_vals[np.newaxis, :]
         model.estimator.coef_ = weight_vals
```

Comment:
Probably yes. Loading pre-trained models for new data is tricky in general:
- if the data distribution is substantially different than what the loaded model trained on we are in dangerous territory independently of what scaler we use.
- if the data distribution is similar, the StandardScaler will come to similar mean and std estimates than the one used for training the loaded model. In that case, if we have an issue it should be minor. Instead, using the scaler="as-is" initializes the DummyScaler, which does not scale. This will likely cause bigger issues than using the StandardScaler with defaults.

Does this make sense? 

I agree with you that loading the scaler in addition to the model would be the optimal thing to do, but would complicate the serialization and loading. Would it be ok to factor this out into a separate issue  / PR?

Reply:
https://github.com/wfondrie/mokapot/pull/119#issuecomment-2231943971 <- Added there to the series of things that should be done but not now